davidstosik/moneyforward_ynab
# Major refactor

- Extract MoneyForward download logic
- Extract MoneyForward CSV reading logic
- Extract logic to import transactions into YNAB
- Remove the concept of "steps".
- Use a temporary directory to save files for now
- Refactor `MoneyForwardDownloader` for _some_ readability
- Remove done task from readme
- Minor refactor of MoneyForwardData
- Switch from YNAB::API to YNAB::ApiClient
- Get started with testing
- Add Rake and Rakefile to run tests
- Set up Bundler when running bin/mfynab
- Update todo list
- Download MoneyForward CSV files with `Net::HTTP`
- Test `MoneyForward#download_csv` happy path
- Remove day from date in CSV file names
- Add some debug print when sending transactions to YNAB


Check individual commit messages for more details.
